src/Database/Migrations/2021-07-04-041948_CreateSettingsTable.php

```diff
@@ -39,6 +51,11 @@ public function up()
         $this->forge->createTable(config('Settings')->database['table'], true);
     }
 
+    private function _getConfig()
```

Comment:
Remove `_` at the beginning.
You don't need to put it.

I recommend to remove this private method.
You can use `config('Settings'` in the constructor.